messaging/relayer/internal/logging/logging_test.go

```diff
@@ -0,0 +1,119 @@
+/*
+Package logging - common package, used for logging purposes. Is a wrapper around 3-rd party logging framework.
+*/
+package logging
+
+/*
+ * Copyright 2021 ConsenSys Software Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import (
+	"fmt"
+
+	"github.com/rs/zerolog"
+	"github.com/spf13/viper"
+)
+
+func ExampleInit() {
+	conf := viper.New()
+	l := Init(conf)
+	fmt.Printf("%T\n", l)
+	// Output: zerolog.Logger
+}
+
+func Example_setTimeFormat_RFC() {
+	conf := viper.New()
+	conf.Set("LOG_TIME_FORMAT", "RFC3339")
+	setTimeFormat(conf)
+	fmt.Println(zerolog.TimeFieldFormat)
+	// Output: 2006-01-02T15:04:05Z07:00
+}
+
+func Example_setTimeFormat_Unix() {
+	conf := viper.New()
+	conf.Set("LOG_TIME_FORMAT", "Unix")
+	setTimeFormat(conf)
+	fmt.Println(zerolog.TimeFieldFormat)
+	// Output:
+}
+
+func ExampleInfo() {
+	Trace("trace")
+	Debug("debug")
+	Info("info")
+	Warn("warn")
+	Error("error")
+	// Output:
+}
+
+func Example_setLogLevel() {
+	conf := viper.New()
+	conf.Set("LOG_LEVEL", "AA")
+	setLogLevel(conf)
```

Comment:
is there a specific purpose for lines 62 and 63 as part of this test? Is the intent to show that the subsequent `setLogLevel()` would override this setting? If so, might it make sense to instead use a valid log level that would not be ignored in place of "AA" e.g. `info` .. alternatively, if the test is to ensure that an unknown level would be ignored, should we add a check for that?

Reply:
Fixed the test.